.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -62,13 +55,15 @@ Release of binaries on conda-forge
 To ensure CADET-Core is accessible to a broad community, it is available as a Python package on conda-forge.
 Other software, such as our frontend, `CADET-Process`, and our Python interface, `CADET-Python`, import this package.
 
-- [ ] Go to github.com/conda-forge/cadet-feedstock
-- [ ] Create a new branch and open a PR:
+- [ ] Go to your fork of github.com/conda-forge/cadet-feedstock or create the fork
+- [ ] Create a new branch on your fork and open a PR:
```

Comment:
In theory, that's not even necessary. You could modify that on any branch on your fork and then use that as base for the PR